gettsim/tests/test_arbeitslosengeld_ui.py

```diff
@@ -49,5 +49,5 @@ def test_ui(input_data, tax_policy_data, year):
         out_cols=[OUT_COL],
         func_kwargs={"tb": tb},
     )
-    # TODO: THis should be reviewed.
+
```

Comment:
Just add a comment, that we verified the correctness and this is due to rounding in ods!